universal-login-wallet/cypress/integration/walletRoundTrip.test.ts

```diff
@@ -21,6 +22,14 @@ describe('E2E: Wallet roundtrip', () => {
       cy.selectCrypto();
       cy.receiveCrypto();
       cy.goToFunds();
+      cy.goToSend().then(() => {
+        address = JSON.parse(localStorage.getItem('wallet-ganache')).wallet.contractAddress;
```

Comment:
Why we are sending money to us? We should send some money to any other address (not ours).